soc/arm/nxp_imx/rt6xx/Kconfig.soc

```diff
@@ -14,13 +14,13 @@ config SOC_MIMXRT685S_CM33
 	select CPU_HAS_FPU
 	select ARMV8_M_DSP
 	select ARM_TRUSTZONE_M
-	select CPU_CORTEX_M_HAS_SYSTICK
```

Comment:
Don't remove this, without it we can't ever use SYSTICK

Reply:
Sure, thank you for spotting this.